docs/UserGuide.md

```diff
@@ -341,7 +341,7 @@ Format: `archive-list`
 
 <div markdown="span" class="alert alert-primary">:information_source: **Note:**
 <br>
-The archive list is a purely read-only list. You can only manipulate it using the command `unarchive`.
+The archive list is a purely read-only list. You can only use two commands while viewing it: `unarchive` (to unarchive a client) and `list` (to navigate back to the main client list).
```

Comment:
we are able to use exit and help command also

Reply:
By any chance would it be worth stating that to delete or edit these clients you have to move them back to the main list?

Reply:
We can technically also use `archive-list` I just realised; should I add that as well?

Also we can put that in future enhancements that you need to move them back to the main list before editing/deleting, and that we plan to allow it soon.